courses/free-courses-pt_BR.md

```diff
@@ -204,6 +204,7 @@
 
 ### Machine Learning
 
+* [Curso Deep Learning](https://www.youtube.com/playlist?list=PLSZEVLiOtIgF19_cPrvhJC2bWn-dUh1zB) - Deep Learning Brasil (Em português) (YouTube)
```

Comment:
There are a problem with `(Em português)` as part of the author. Conflicts with resources format syntax & metadata meaning 

options: remove or escape

@eshellman, What do you think about?

Reply:
can I remove the "(" ?

Reply:
It's the portuguese list. Of course it's Em português!